src/middlewared/middlewared/rclone/remote/storjix.py

```diff
@@ -0,0 +1,55 @@
+import io
+import textwrap
+import xml.etree.ElementTree as ET
+
+from aws_requests_auth.aws_auth import AWSRequestsAuth
+import requests
+
+from middlewared.rclone.base import BaseRcloneRemote
+from middlewared.schema import Int, Str
+from middlewared.utils.network import INTERNET_TIMEOUT
+
+
+class StorjIxRcloneRemote(BaseRcloneRemote):
+    name = "STORJ_IX"
+    title = "Storj iX"
+
+    buckets = True
+    custom_list_buckets = True
+
+    fast_list = True
+
+    rclone_type = "s3"
+
+    credentials_schema = [
+        Str("access_key_id", title="Access Key ID", required=True),
+        Str("secret_access_key", title="Secret Access Key", required=True),
+    ]
+
+    task_schema = []
+
+    async def list_buckets(self, credentials):
```

Comment:
Your outer and inner methods are named the same. Please change name of inner method to something else or move it into its own method.

Reply:
Will do that, but just curious why can't they be names the same?

Reply:
Just seems strange to name it the same. When I see methods that are named the same, I'm thinking of method overwriting when inheriting from a parent or super class.